app/views/content/service_guidance_provider.md

```diff
@@ -84,6 +84,57 @@ Users in your team will be able to:
 * withdraw
 * reject, with reasons
 
+###Setting permissions for your team members
+
+The permissions feature allows you to control:
+
+* who can make decisions about applications
+* who can view sensitive information disclosed by the candidate (in the safeguarding or equality and diversity sections of the application)
+* who can invite other users, and set their permissions
+
+For example, you may wish to:
+
+* give members of your admission team access to all applications, but not the ability to make decisions
+* give academic decision-makers access to all applications, but not to sensitive information
+* limit the number of people with control of these permissions
+
+Everyone on your team who uses the Manage teacher training applications service will be able to view all applications your organisation is involved with. You do not need to set permissions for this.
+
+###Permissions for training providers who are already part of the Manage teacher training pilot
+
+We have automatically given the person in your organisation who signed the data-sharing agreement ‘Manage users’ permission. (If you’re not sure who signed the agreement, get in touch with us on becomingateacher@digital.education.gov.uk.)
+
+###Permissions for training providers now being onboarded
+
+As part of your onboarding, we asked you to nominate team members to fill the ‘Manage users’ role. (Get in touch with us on <becomingateacher@digital.education.gov.uk> if you've yet to nominate a person for this role.)
+
+‘Manage users’ permission allows the user to:
+
+* set permissions for your team members
+* invite new users
+* delete users
+
+You can delete a user’s ‘Manage users’ permission and assign it to a different person if you need to.
+
+###Default settings and how to change them
+
+We’ve created some default settings for the service:
```

Comment:
These are settings for existing users — newly created users will need make_decisions to be explicitly set, it won't be set by default